Makefile

```diff
@@ -39,6 +43,10 @@ cover-viz: cover
 build: vendor
 	go build -tags release -ldflags $(LDFLAGS) $(FLAGS) ./cmd/pilosa
 
+# Compile Pilosa EE
+build-ee: vendor
```

Comment:
what about install-ee?

Reply:
I am changing the Makefile now so it will use the same make commands but with an env flag (ie. `make install ENTERPRISE=1`).